runtime/common/src/lib.rs

```diff
@@ -146,6 +152,30 @@ pub mod types {
 	)]
 	#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 	pub struct InstanceId(pub u128);
+
+	bitflags::bitflags! {
+		/// The current admin roles we support
+		#[derive(codec::Encode, codec::Decode,  TypeInfo)]
+		pub struct AdminRoles: u32 {
+			const POOL_ADMIN = 0b00000001;
+			const BORROWER  = 0b00000010;
+			const PRICING_ADMIN = 0b00000100;
+			const LIQUIDITY_ADMIN = 0b00001000;
+			const MEMBER_LIST_ADMIN = 0b00010000;
+			const RISK_ADMIN = 0b00100000;
```

Comment:
does this mean we cannot have more than 8 roles defined. or we can we extend to 16bit without any migration later in case the situation arises ?

Reply:
The `AdminRoles`supports 32 roles without another storage migration.